client/app/common/models/Environment.js

```diff
@@ -10,7 +10,8 @@ angular.module('EnvironmentManager.common').factory('Environment',
 
     Environment.all = function (params) {
       var query = params && params.query;
-      return $http.get('/api/v1/config/environments', { params: query, cache: true }).then(function (response) {
+      var useCache = !params || (params.useCache !== false);
```

Comment:
If we used ```params !== undefined``` and someone did pass one of the other falsey values (false, 0, "", null, NaN), it would evaluate to true, and we would then attempt to get params.useCache which would then throw an exception. This is much less safe. I supposed I could use _.isObject